sonar-db/src/main/java/org/sonar/db/version/v53/FixMsSqlCollation.java

```diff
@@ -0,0 +1,553 @@
+      .addVarcharColumn("kee", 100)
+      .addVarcharColumn("name", 100)
+      .addVarcharColumn("function_key", 100)
+      .addVarcharColumn("factor_unit", 100)
+      .addVarcharColumn("offset_unit", 100)
+      .execute();
+
+    // rules_parameters
+    new UpdateTableCollation(context, db, "rules_parameters")
+      .addNotNullableVarcharColumn("name", 128)
+      .addNotNullableVarcharColumn("param_type", 512)
+      .addVarcharColumn("default_value", 4000)
+      .addVarcharColumn("description", 4000)
+      .execute();
+
+    // rules_profiles
+    new UpdateTableCollation(context, db, "rules_profiles")
+      .addUniqueIndex("uniq_qprof_key", "kee")
+      .addNotNullableVarcharColumn("name", 100)
+      .addVarcharColumn("language", 20)
+      .addNotNullableVarcharColumn("kee", 255)
+      .addVarcharColumn("parent_kee", 255)
+      .addVarcharColumn("rules_updated_at", 100)
+      .execute();
+
+    // project_qprofiles
+    new UpdateTableCollation(context, db, "project_qprofiles")
+      .addUniqueIndex("uniq_project_qprofiles", "project_uuid", "profile_key")
+      .addNotNullableVarcharColumn("project_uuid", 50)
+      .addNotNullableVarcharColumn("profile_key", 255)
+      .execute();
+
+    // widgets
+    new UpdateTableCollation(context, db, "widgets")
+      .addIndex("widgets_widgetkey", "widget_key")
+      .addNotNullableVarcharColumn("widget_key", 256)
+      .addVarcharColumn("name", 256)
+      .addVarcharColumn("description", 1000)
+      .execute();
+
+    // groups
+    new UpdateTableCollation(context, db, "groups")
+      .addVarcharColumn("name", 500)
+      .addVarcharColumn("description", 200)
+      .execute();
+
+    // snapshots
+    new UpdateTableCollation(context, db, "snapshots")
+      .addIndex("snapshots_qualifier", "qualifier")
+      .addVarcharColumn("scope", 3)
+      .addVarcharColumn("qualifier", 10)
+      .addVarcharColumn("version", 500)
+      .addVarcharColumn("path", 500)
+      .execute();
+
+    // schema_migrations
+    new UpdateTableCollation(context, db, "schema_migrations")
+      .addIndex("unique_schema_migrations", "version")
+      .addNotNullableVarcharColumn("version", 256)
+      .execute();
+
+    // group_roles
+    new UpdateTableCollation(context, db, "group_roles")
+      .addUniqueIndex("uniq_group_roles", "group_id", "resource_id", "role")
+      .addIndex("group_roles_role", "role")
+      .addNotNullableVarcharColumn("role", 64)
+      .execute();
+
+    // Rules
+    new UpdateTableCollation(context, db, "rules")
+      .addUniqueIndex("rules_repo_key", "plugin_name", "plugin_rule_key")
+      .addNotNullableVarcharColumn("plugin_rule_key", 200)
+      .addNotNullableVarcharColumn("plugin_name", 255)
+      .addClobColumn("description")
+      .addVarcharColumn("description_format", 20)
+      .addVarcharColumn("plugin_config_key", 500)
+      .addVarcharColumn("name", 200)
+      .addVarcharColumn("status", 40)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "status" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)